content/en/cypherpunk/hal-finney/politics-vs-technology.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Politics vs Technology"
+description: "La crittografia sarà sufficiente a garantire la privacy? O continuerà a essere necessaria una lotta politica? - 2 Gennaio, 1994"
+date: 2023-01-29T17:54:11+01:00
+draft: false
+weight: 320
+images: []
+categories: []
+contributors: []
+---
+
+[Originale](https://web.archive.org/web/20041206203625/http://finney.org:80/~hal/pol_v_tech.html) - 2 Gennaio, 1994
+
+Ma il governo potrebbe vietare un libro oggi? Certamente no, almeno non dopo che una persona l'ha digitato o scannerizzato in un computer. I successi tecnologici sono permanenti. L'approccio politico è utile solo come arma tattica, per frenarli fino a quando non saranno disponibili soluzioni tecnologiche. Se vuoi cambiare il mondo, non protestare. Scrivi codice!
+
+Questa posizione sembra essere diventata rapidamente un dogma cypherpunks, ma io non sono d'accordo. L'idea che possiamo semplicemente svanire nel cypherspace e ignorare le spiacevoli realtà politiche è irrealistica, a mio avviso.
+
+Le persone hanno dimenticato la proposta Clipper, con la possibile conseguenza di rendere illegale la crittografia non Clipper? Nel caso in cui questa proposta sia stata o sarà sconfitta, ciò avverrà grazie alle manovre politiche, non alla tecnologia.
+
+Le persone hanno dimenticato l'indagine sull'esportazione di PGP? Phil Zimmermann no. Lui e altri potrebbero trovarsi di fronte alla prospettiva di dieci anni di prigione se venissero riconosciuti colpevoli di esportazione illegale. Se qualcuno ha qualche suggerimento su come fuggire dalla prigione nel cyberspazio, mi piacerebbe sentirlo.
+
+SecureDrive di Mike è un programma straordinario per proteggere la privacy. Ma se vogliamo mantenere le chiavi segrete da indagini di natura politica, dobbiamo affidarci al Quinto Emendamento, molto politico e non tecnologico (un emendamento che Mike Godwin dell'EFF e altri sostengono non protegga effettivamente la divulgazione delle chiavi crittografiche). Ancora una volta, dobbiamo ottenere vittorie politiche, non tecnologiche, per proteggere la nostra privacy.
+
+Metto anche in dubbio il punto di Mike sull'incapacità del governo di vietare i libri. Guarda la difficoltà di mantenere il PGP disponibile in questo Paese, anche se è legale. Non solo i siti FTP sono stati costantemente chiusi, ma anche le chiavi server. E questo è un software legale.
+
+Certo, questo software è attualmente disponibile all'estero, ma questo perché le uniche limitazioni legali del PGP sono i problemi di brevetto degli Stati Uniti. Immagina come sarebbe peggio se la crittografia non escrowed fosse resa illegale in un'ampia gamma di Paesi, con limiti stringenti all'accesso alla rete nei Paesi che promuovono il software illegale? Anche in questo caso, questo tipo di decisioni saranno prese in ambito politico.
```

Comment:
Io propongo così: "Immagina quanto peggio ancora sarebbe se la crittografia non vincolata fosse resa illegale in un ampia gamma di paesi, con limiti rigorosi sull'accesso alla rete verso paesi che promuovono software illegale?"